WMass/python/plotter/results_config_lep.txt

```diff
@@ -4,17 +4,17 @@ fitsdir_el = '/afs/cern.ch/work/e/emanuele/wmass/heppy/CMSSW_8_0_25/src/CMGTools
 fitsdir_mu = '/afs/cern.ch/work/e/emanuele/wmass/heppy/CMSSW_8_0_25/src/CMGTools/WMass/python/plotter/w-helicity-13TeV/fitsmu/barolo-decfsr/fitsout_2019-11-08/'
 
 ### FLOATING POIS ###
-both_floatingPOIs_hessian_bbb1_syst1_asimov     = fitsdir+   '/fitresults_123456789_poim1_exp1_bbb1.root'
-both_floatingPOIs_hessian_bbb1_syst1_asimov_el = fitsdir_el+'/fitresults_123456789_poim1_exp1_bbb1.root'
-both_floatingPOIs_hessian_bbb1_syst1_asimov_mu = fitsdir_mu+'/fitresults_123456789_poim1_exp1_bbb1.root'
+#both_floatingPOIs_hessian_bbb1_syst1_asimov     = fitsdir+   '/fitresults_123456789_poim1_exp1_bbb1.root'
+#both_floatingPOIs_hessian_bbb1_syst1_asimov_el = fitsdir_el+'/fitresults_123456789_poim1_exp1_bbb1.root'
+#both_floatingPOIs_hessian_bbb1_syst1_asimov_mu = fitsdir_mu+'/fitresults_123456789_poim1_exp1_bbb1.root'
 
 both_floatingPOIs_hessian_bbb1_syst1_data     = fitsdir+   '/fitresults_123456789_poim1_exp0_bbb1.root'
 both_floatingPOIs_hessian_bbb1_syst1_data_el = fitsdir_el+'/fitresults_123456789_poim1_exp0_bbb1.root'
 both_floatingPOIs_hessian_bbb1_syst1_data_mu = fitsdir_mu+'/fitresults_123456789_poim1_exp0_bbb1.root'
 
 ### FIXED POIS ###
-both_fixedPOIs_hessian_bbb1_syst1_asimov = fitsdir+'/fitresults_123456789_poim0_exp1_bbb1.root'
-both_fixedPOIs_hessian_bbb1_syst1_data   = fitsdir+'/fitresults_123456789_poim0_exp0_bbb1.root'
+#both_fixedPOIs_hessian_bbb1_syst1_asimov = fitsdir+'/fitresults_123456789_poim0_exp1_bbb1.root'
```

Comment:
same here

Reply:
It was because I didn't want to make all those plots, I only needed the plots for combination on data.
Then I forgot to decomment